frontend/survey-project/src/app/components/shared/ui/footer/footer.component.css.map

```diff
@@ -0,0 +1 @@
+{"version":3,"sources":["footer.component.scss","footer.component.css"],"names":[],"mappings":"AAAA;EACI,iBAAA;EACA,yBAAA;EACA,wDAAA;EACA,iBAAA;EACA,WAAA;EACA,4BAAA;EACA,yCAAA;EACA,kBAAA;EACA,WAAA;EACA,SAAA;ACCJ;ADCI;EACI,mBAAA;ACCR;;ADII;EACI,aAAA;EACA,sBAAA;EACA,uBAAA;EACA,qBAAA;EACA,UAAA;EACA,SAAA;ACDR;ADGQ;EACI,aAAA;EACA,mBAAA;EACA,mBAAA;EACA,mBAAA;ACDZ;ADGY;EACI,aAAA;EACA,mBAAA;EACA,mBAAA;EACA,qBAAA;EACA,WAAA;EACA,eAAA;EACA,gBAAA;EACA,SAAA;ACDhB;ADGgB;EACI,WAAA;EACA,YAAA;EACA,sBAAA;KAAA,mBAAA;ACDpB;ADOI;EACI,aAAA;EACA,sBAAA;EACA,mBAAA;EACA,SAAA;EACA,gCAAA;EACA,+BAAA;EACA,+BAAA;EACA,yBAAA;EACA,kBAAA;EACA,eAAA;EACA,WAAA;EACA,qBAAA;ACLR;ADOQ;EACI,YAAA;EACA,aAAA;EACA,sBAAA;KAAA,mBAAA;ACLZ;ADSI;EACI,aAAA;EACA,sBAAA;EACA,eAAA;ACPR;ADUI;EACI,kBAAA;EACA,eAAA;EACA,gBAAA;ACRR","file":"footer.component.css"}
```

Comment:
Not sure what is this used for, but I think it's not needed, we're using scss not css and the Angular builder will build our scss to css.